js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -100,7 +103,7 @@ class RequestsPage extends Component {
           <div className={ styles.noRequestsMsg }>
             <FormattedMessage
               id='signer.requestsPage.noPending'
-              defaultMessage='There are no requests requiring your confirmation.'
+              defaultMessage='There are no transactions requiring your confirmation.'
```

Comment:
I like the simple approach or we can use the opportunity to explain why there is a Signer in the first place :)